climada/hazard/tc_clim_change.py

```diff
@@ -29,88 +29,124 @@
 generated: 2018-07-04 10:47:59."""
 
 def get_knutson_criterion():
-    """Fill changes in TCs according to Knutson et al. 2015 Global projections
+    """
+    Fill changes in TCs according to Knutson et al. 2015 Global projections
     of intense tropical cyclone activity for the late twenty-first century from
     dynamical downscaling of CMIP5/RCP4.5 scenarios.
 
-    Returns:
-        list(dict) with items 'criteria' (dict with variable_name and list(possible values)),
-        'year' (int), 'change' (float), 'variable' (str), 'function' (np function)
+    Returns
+    -------
+    criterion : list(dict)
+        list of the criterion dictionary for frequency and intensity change
+        per basin, per category taken from the Table 3 in Knutson et al. 2015.
+        with items 'basin' (str), 'category' (list(int)), 'year' (int),
+        'change' (float), 'variable' ('intensity' or 'frequency')
     """
     criterion = list()
     # NA
-    tmp_chg = {'criteria': {'basin': ['NA'], 'category': [1, 2, 3, 4, 5]},
-               'year': 2100, 'change': 1.045, 'variable': 'intensity', 'function': np.multiply}
+    tmp_chg = {'basin': 'NA', 'category': [0, 1, 2, 3, 4, 5],
+               'year': 2100, 'change': 1, 'variable': 'frequency'}
+    criterion.append(tmp_chg)
+    tmp_chg = {'basin': 'NA', 'category': [1, 2, 3, 4, 5],
+               'year': 2100, 'change': 1, 'variable': 'frequency'}
+    criterion.append(tmp_chg)
+    tmp_chg = {'basin': 'NA', 'category': [3, 4, 5],
+               'year': 2100, 'change': 1, 'variable': 'frequency'}
+    criterion.append(tmp_chg)
+    tmp_chg = {'basin': 'NA', 'category': [4, 5],
+               'year': 2100, 'change': 1, 'variable': 'frequency'}
+    criterion.append(tmp_chg)
+
+    tmp_chg = {'basin': 'NA', 'category': [1, 2, 3, 4, 5],
+               'year': 2100, 'change': 1.045, 'variable': 'intensity'}
     criterion.append(tmp_chg)
```

Comment:
Thanks for cleaning up a bit. While you're at it, can you maybe remove all the annoying append-operations?
```suggestion
    criterion = [
        {'basin': 'NA', 'category': [0, 1, 2, 3, 4, 5],
         'year': 2100, 'change': 1, 'variable': 'frequency'},
        {'basin': 'NA', 'category': [1, 2, 3, 4, 5],
         'year': 2100, 'change': 1, 'variable': 'frequency'},
        {'basin': 'NA', 'category': [3, 4, 5],
         'year': 2100, 'change': 1, 'variable': 'frequency'},
        {'basin': 'NA', 'category': [4, 5],
         'year': 2100, 'change': 1, 'variable': 'frequency'},
        {'basin': 'NA', 'category': [1, 2, 3, 4, 5],
         'year': 2100, 'change': 1.045, 'variable': 'intensity'},
         # and so on for all the criteria in the function ...
         ]
```

Reply:
> Thanks for cleaning up a bit. While you're at it, can you maybe remove all the annoying append-operations?

Excellent idea!